src/ts/components/subtitlesettings/settinglist.ts

```diff
@@ -0,0 +1,31 @@
+import {FontColorSelectBox} from './fontcolorselectbox'
+import {FontOpacitySelectBox} from './fontopacityselectbox'
+import {FontFamillySelectBox} from './fontfamillyselectbox'
+import {FontSizeSelectBox} from './fontsizeselectbox'
+import {BackgroundColorSelectBox} from './backgroundcolorselectbox'
+import {BackgroundOpacitySelectBox} from './backgroundopacityselectbox'
+import {WindowColorSelectBox} from './windowcolorselectbox'
+import {WindowOpacitySelectBox} from './windowopacityselectbox'
+import {CharacterEdgeSelectBox} from './characteredgeselectbox'
+import {SettingsPanelItem} from '../settingspanel'
+import {SubtitleOverlay} from '../subtitleoverlay'
+import {SubtitlePanelCloser} from '../subtitlesettingtoggle'
+import {Component, ComponentConfig} from '../component'
+
+// Helper function since the subtitle option panel should not change
+let GetSubtitleSettingList = (overlay: SubtitleOverlay): Component<ComponentConfig>[] => {
+    return [
+        new SettingsPanelItem('Font', new FontColorSelectBox({}, overlay)),
+        new SettingsPanelItem('Font opacity', new FontOpacitySelectBox({}, overlay)),
+        new SettingsPanelItem('Background', new BackgroundColorSelectBox({}, overlay)),
```

Comment:
Ok, for the name change. 

@protyposis Background and Window are two different things, the window is the overall box, containing the captions, whereas background is just "behind" the text. It's explicitly mentioned by the FCC doc.

![background-window](https://user-images.githubusercontent.com/8531875/27123676-e7af6e2c-50ee-11e7-9fea-47a23601f567.png)

@dweinber what do you mean grouping them? Having a sub menu for each?